resources/assets/lib/beatmap-discussions/editor-discussion-component.tsx

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+import * as React from 'react';
+import { Path, Transforms } from 'slate';
+import { RenderElementProps } from 'slate-react';
+import { ReactEditor } from 'slate-react';
+import { BeatmapsetJson } from '../beatmapsets/beatmapset-json';
+import BeatmapJsonExtended from '../interfaces/beatmap-json-extended';
+import EditorBeatmapSelector from './editor-beatmap-selector';
+import EditorIssueTypeSelector from './editor-issue-type-selector';
+import { SlateContext } from './slate-context';
+
+interface Props extends RenderElementProps {
+  // attributes taken from RenderElementProps, but extended with contentEditable
+  attributes: {
```

Comment:
the comment is now rather confusing because it's not extending anything anymore...

Remove?